src/app/static/templates/login.ftl

```diff
@@ -50,7 +54,7 @@
     </div>
     <div class="text-center mt-3">
         <div class="mb-3">OR</div>
-        <a class="btn btn-red" type="submit" href="/">Continue as guest</a>
+        <a class="btn btn-red" onclick="continueAsGuest()" type="submit" href="/">Continue as guest</a>
```

Comment:
One thing you could do is check that this attribute is present in [LoginTests.kt](https://github.com/mrc-ide/hint/blob/master/src/app/src/test/kotlin/org/imperial/mrc/hint/unit/templates/LoginTests.kt#L23)